gravitee-apim-console-webui/src/management/api/creation-v4/steps/step-4-security/step-4-security-1-plans.harness.ts

```diff
@@ -121,11 +122,12 @@ export class Step4Security1PlansHarness extends ComponentHarness {
     apiPlanFormHarness.httpRequest(httpTestingController).expectGroupLisRequest([fakeGroup({ id: 'group-a', name: 'Group A' })]);
 
     await (await this.getButtonByText('Next')).click();
-
     const rateLimitToggle = await apiPlanFormHarness.getRateLimitEnabledInput();
     await rateLimitToggle.toggle();
 
-    apiPlanFormHarness.httpRequest(httpTestingController).expectPolicySchemaGetRequest('rate-limit', {});
+    // When form has initial value, value is patched and multiple requests can be sent to get the schema. Need to use match here, but we check that at least one request is done
+    const req = httpTestingController.match({ url: `${CONSTANTS_TESTING.env.baseURL}/policies/rate-limit/schema`, method: 'GET' });
+    expect(req.length > 0);
```

Comment:
I remember I had the same issue on another test. I realized that I could ignore first requests as they were "cancelled".
https://github.com/gravitee-io/gravitee-api-management/blob/master/gravitee-apim-console-webui/src/management/api/general/subscriptions/list/api-portal-subscription-list.component.spec.ts#L846-L854

Maybe it is the same for you here?

Reply:
You can also configure the httpTestController to ignore cancelled calls if that's the case